packages/contracts-bedrock/scripts/DeployConfig.s.sol

```diff
@@ -110,6 +112,8 @@ contract DeployConfig is Script {
         sequencerFeeVaultRecipient = stdJson.readAddress(_json, "$.sequencerFeeVaultRecipient");
         sequencerFeeVaultMinimumWithdrawalAmount = stdJson.readUint(_json, "$.sequencerFeeVaultMinimumWithdrawalAmount");
         sequencerFeeVaultWithdrawalNetwork = stdJson.readUint(_json, "$.sequencerFeeVaultWithdrawalNetwork");
```

Comment:
If you could add a `useRevenueShare = _envOr(_json, ".useRevenueShare", true);` that would be great then in the Genesis generation you can know to configure the fee vaults appropriately when its set to true. Will need to add a setter `setUseRevenueShare` to be able to not break legacy tests, you can see how this is done with plasma, you need to call it in the tests `setUp` before calling `super.setUp`